druid-shell/src/gtk/util.rs

```diff
@@ -0,0 +1,23 @@
+// Copyright 2019 The xi-editor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Utilities, GTK specific.
+
+pub fn init() {
+    gtk::init().expect("GTK initialization failed");
+}
```

Comment:
I don't think you're using this and you don't have to since creating the Application will do this for you.